tests/test_cli.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from typer.testing import CliRunner
+
+from minari.cli import app
+from minari.storage.local import delete_dataset, list_local_datasets
+from tests.dataset.test_dataset_download import get_latest_compatible_dataset_id
+
+
+runner = CliRunner()
+
+
+def test_list_app():
+    result = runner.invoke(app, ["list", "local", "--all"])
+    assert result.exit_code == 0
+    # some of the other columns may be cut off by Rich
+    assert "Name" in result.stdout
+
+    result = runner.invoke(app, ["list", "remote"])
+    assert result.exit_code == 0
+    assert "Minari datasets in Farama server" in result.stdout
+
+
+@pytest.mark.parametrize(
+    "dataset_id",
+    [get_latest_compatible_dataset_id(env_name="pen", dataset_name="human")],
+)
+def test_dataset_download_then_delete(dataset_id: str):
+    """test download dataset invocation from CLI.
+    the downloading functionality is already tested in test_dataset_download.py so this is primarily to assert that the CLI is working as expected.
+    """
+
+    # might have to clear up the local dataset first.
+    # ideally this seems like it could just be handled by the tests
```

Comment:
It might be possible to mock the actual dataset download function called by the cli function if we really want to avoid repetition, particularly because dataset downloads are slow and add to the overall test time, but I don't think it's strictly speaking required.

Reply:
these lines + 34+35 probably can be removed but i dont know if some other test will download the dataset for testing elsewhere and this runs concurrent or afterwords and the dataset wasnt cleaned up by the other test.

Reply:
As long as each test cleans up after itself, its fine if they also have redundant cleaning at the beginning. Your test cleans up the dataset it downloads, so I think it's fine.